src/lt/plugins/clojure.cljs

```diff
@@ -643,16 +708,18 @@
                           (object/raise editor :editor.doc.show! result)))))
 
 (behavior ::clj-doc-search
+          :doc "Links the 'Search language docs' input-text on the sidebar with a trigger to
```

Comment:
Please convert to :desc. While I'm ok using :doc as an additive key we shouldn't be using it as a replacement for :desc. Same feedback for other behaviors that only have :doc